README.md

```diff
@@ -126,24 +126,27 @@ Secrets().getYourSecretKeyName(packageName)
 ### Copy files
 Copy required files to your project :
 ```shell
-gradle copyCpp
-gradle copyKotlin [-Ppackage=your.package.name]
+./gradlew copyCpp
+./gradlew copyKotlin [-Ppackage=your.package.name]
 ```
 
 ### Obfuscate
 Create an obfuscated key and display it :
 ```shell
```

Comment:
Gradle is supported by syntax engine.

```suggestion
``` gradle
```

Reply:
@Mazorius I used `gradle` in the beginning of the readme for gradle files. This is a command line what will be the difference by annotating it as a gradle despite it is a shell/console command ?

Reply:
Sorry - was my bad. gradle is for Gradle files and not shell commands.
You can ignore my suggestions 🤦